mesos/src/main/java/brooklyn/location/mesos/framework/marathon/MarathonLocation.java

```diff
@@ -161,6 +161,7 @@ private String getTaskName(Entity entity) {
         String planId = entity.config().get(BrooklynCampConstants.PLAN_ID);
```

Comment:
and could reuse

    DockerUtils.getEntitiyIdentifierForContainerName(Entity target, String separator)

based on `getContainerNameFromPlan`